modules/components/src/Arranger/CurrentSQON.js

```diff
@@ -35,10 +35,9 @@ export const CurrentSQON = ({
             translateSQONValue,
             internalTranslateSQONValue,
           )(
-            (findExtendedMappingField(field)?.displayValues || {})[value] ||
-              findExtendedMappingField(field)?.type === 'date'
```

Comment:
this || was all bundled up as the single condition for the ternary, causing boolean values to be misidentified as dates